### PR TITLE
User authentication demo in nodejs hapi framework

### PR DESCRIPTION
**Pull request by karmanya007**
#### Checklist

- [x] You have read the [Contribution Guidlines](https://github.com/LoginRadius/developer-authentication-demos/blob/master/guidelines.md) before creating this PR.
- [x] This is not a duplicate entry for any framework or language\

##### Description of change

Created a user authentication demo in nodejs hapi framework.

### CODE REVIEW

File: demos/nodejs/hapi/config.js.sample [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const config = {
+  apiDomain: 'api.loginradius.com',
+  apiKey: '<API_KEY>',
+  apiSecret: '<API_SECRET'>,
+  siteName: <APP_NAME'>',
```
Comment (pareek-narendra) [inline demos/nodejs/hapi/config.js.sample:5]: @karmanya007 
Please correct the misplaced single quotes in this and express folder.

**Review by pareek-narendra — CHANGES_REQUESTED** [review]

File: demos/nodejs/hapi/server.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        host: 'localhost',
+        routes: {
+            files: {
+                relativeTo: VIEW_PATH
+            }
+        }
+    });
+
+    await server.register(require('@hapi/inert'));
+    server.route({
```
Comment (pareek-narendra) [inline demos/nodejs/hapi/server.js:23]: @karmanya007 It would be better if you can add comments for routes.

**Review by pareek-narendra — CHANGES_REQUESTED** [review]

File: demos/nodejs/hapi/server.js (hunk 2 above)
Comment (karmanya007) [reply demos/nodejs/hapi/server.js:23]: Okay.

**Review by karmanya007 — COMMENTED** [review]
Added comments in server.js

**Review by pareek-narendra — APPROVED** [review]

**Pull request merged**